src/bp/core/services/job-service.ts

```diff
@@ -0,0 +1,21 @@
+import { injectable } from 'inversify'
+
+export interface JobService {
+  /**
+   * * In Botpress Pro, this allows to broadcast a job to execute to all distributed nodes.
+   * It returns a function that returns a promise.
+   * The promise will succeed if all nodes have run the job and will fail otherwise.
+   *
+   * * In Botpress CE, the function will be returned directly.
+   * @param fn The function or "job" to execute
+   * @param jobName The optionnal name of the job. If not specified, only a unique Id will identify the job.
```

Comment:
not sure to me what that does, can you add a comment as to under which circumstances we should provide a jobName?

Reply:
Will do, mostly to debug and provide a meaningful name to the job. I removed the fn.name from the `jobId` because the formatting was messed up. i.e. `bound _mountBot`. Didn't want to format the name because its kinda sketchy.